ifc/templates/party/fraternityPartyList.html

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
It looks like this file isn't being rendered anywhere, did you write this without seeing it?